apps/omg_watcher/test/omg_watcher/exit_processor/core/state_interaction_test.exs

```diff
@@ -179,55 +179,6 @@ defmodule OMG.Watcher.ExitProcessor.Core.StateInteractionTest do
     assert Utxo.position(1, 0, 0) in spends_to_get
   end
 
-  test "can work with State to exit utxos from in-flight transactions",
```

Comment:
There is IFE cabbage test covering IFE flow of startIFE, piggyback, and process Exit